sensor/common/centralclient/grpc_connection.go

```diff
@@ -43,21 +42,15 @@ func NewCentralConnectionFactory(centralClient *Client) CentralConnectionFactory
 
 // OkSignal returns a concurrency.Signal that is sends signal once connection object is successfully established
```

Comment:
```suggestion
// OkSignal returns a concurrency.ReadOnlySignal that is sends signal once connection object is successfully established
```
Might want to update the doc string if we change the return

Reply:
Good catch! It should be fixed in the last commit